CHANGELOG.md

```diff
@@ -3,6 +3,11 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+- Lazy-load the dependencies that are not used by the browser code.
+- **_Potentially_ breaking!** `bluebird` is no longer a dependency of the project,
```

Comment:
Yes but I think that's OK as we never said we return the Bluebird promise specifically.